### ISSUE TITLE
Rename

### ISSUE THREAD

**Issue by meagan7** [learntocodemeetup/learntocodemeetup.github.io#1]
Hey Lukas, thanks for the upload, could we rename this to Star Wars Character Selection Screen or something like that?

**Comment by LukasTomek** [comment]
We can but it wouldn't be working like web page. You can see: https://pages.github.com/

**Comment by danielalbertjeffery** [comment]
Hey Lukas,

I think she is asking if you can rename the name of the repository!

I'd probably just call it "Star Wars Game" instead of being specific, as
it's likely to expand to be more than just that, and I think we'd probably
want it to be all in the same repository

**Comment by LukasTomek** [comment]
Yes I understood. If you create a repository named username.github.io, where username is your username (or organization name) on GitHub, your repository will work like web page. Check http://learntocodemeetup.github.io/

**Comment by nii236** [comment]
An alternative would be to create a repo with any name, and then create a `gh-pages` branch. The URL will be what the repo name is.

My blog is here: https://github.com/nii236/technical-blog

It contains a branch called `gh-pages`.

This website is accessible at http://nii236.github.io/technical-blog/

**Comment by danielalbertjeffery** [comment]
Ohh alright, I didn't know that, nice :)

**Comment by LukasTomek** [comment]
Ok We can use nii236 advice. If you want do it, otherwise I will do it tomorrow or later.

**Comment by meagan7** [comment]
I didn't mean to complicate things, I think Lukas has overridden our code into the old github page that John made. That's fine. I was only asking to make it more descriptive. No rush Lukas.

**Comment by meagan7** [comment]
at least we know what it looks like on the browser!
